### ISSUE TITLE
Getting error : cannot execute binary file: Exec format error'

### ISSUE THREAD

**Issue by tnshibu** [ipsingh06/seedsync#69]
My local system is a Windows running the seedsync via docker(AMD64 architecture).
My remote system is a raspberry Pi (ARM architecture).

My script location on server is /home/pi/tmp/ and the remote userid is 'pi' . so there is no permission related issue.

I think the issue here is the seedsync software running on AND arch is pushing the scanner script into the RaspberryPi and trying to run the scanner on the server.
Since the architecture on server(ARM32) is different from architecture of client(AMD64), i am getting this "Exec format error"
Can you please have a setting for "Remote server architecture" with options for AMD32, AMD64, ARM32, ARM64, etc...
and then the seedsync can upload the corresponding scanner script to the server?

**Comment by ipsingh06** [comment]
@tnshibu you are correct, only AMD64 is supported for remote server right now. I'm looking into adding support for ARM and ARM64 for remote server as well. Stay tuned.
